packages/webapp/src/routes/orders/[network]-[orderHash]/+page.svelte

```diff
@@ -42,7 +45,28 @@
 			refetchType: 'all',
 			exact: false
 		});
-		triggerToast();
+		triggerToast($transactionStore.message);
+	}
+
+	function handleRemoveOrder(event: CustomEvent<{ order: SgOrder }>) {
```

Comment:
this fn would be better in its own file where it can be tested. it could also be totally pure